create_db.sh

```diff
@@ -26,6 +26,7 @@ sleep 5
 # timeout 90s bash -c "until docker exec $DOCKER_CONTAINER_NAME pg_isready ; do sleep 5 ; done
 
 docker exec -it ${container_id} psql -U ${pg_user} ${pg_db} -f /opt/sql/draft-db.sql
+docker exec ${container_id} pg_isready
```

Comment:
Have a look here https://www.postgresql.org/docs/current/app-pg-isready.html
The command `pg_isready` returns always, but you have to check the exit status to find out if it succeeded. I suggest you use a `while` loop with a `sleep` command in it.